ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4453,30 +4453,30 @@
+		<entry key="TrackElementName_CableTowerRevMid" translation="Sezione Mediana Torre di sollevamento a cavo invertita"/>
 		<entry key="TrackElementName_CableTowerRevStart" translation="Cavo Orizzontale trainante invertito"/>
-		<entry key="TrackElementName_CableTowerRevTop" translation="Cima Torre trainante a cavo invertita"/>
+		<entry key="TrackElementName_CableTowerRevTop" translation="Cima Torre di sollevamento a cavo invertita"/>
 		<entry key="TrackElementName_CableTowerStart" translation="Cavo Orizzontale trainante"/>
-		<entry key="TrackElementName_CableTowerTop" translation="Cima Torre trainante a cavo"/>
+		<entry key="TrackElementName_CableTowerTop" translation="Cima Torre di sollevamento a cavo"/>
 		<entry key="TrackElementName_ChainHillBot" translation="Base Salita trainante a catena"/>
 		<entry key="TrackElementName_ChainHillMid" translation="Sezione Mediana trainante a catena"/>
 		<entry key="TrackElementName_ChainHillTop" translation="Cima Salita trainante a catena"/>
 		<entry key="TrackElementName_ChicaneEnd" translation="Sezione Finale Chicane"/>
-		<entry key="TrackElementName_ChicaneEndRailings" translation="Sezione Finale Parapetti Chicane"/>
+		<entry key="TrackElementName_ChicaneEndRailings" translation="Sezione Finale Chicane con guardrail"/>
 		<entry key="TrackElementName_ChicaneSection" translation="Chicanes"/>
-		<entry key="TrackElementName_ChicaneSectionRailings" translation="Parapetti Chicanes "/>
+		<entry key="TrackElementName_ChicaneSectionRailings" translation="Chicanes con guardrail "/>
 		<entry key="TrackElementName_ChicaneStart" translation="Sezione di Partenza Chicane"/>
-		<entry key="TrackElementName_ChicaneStartRailings" translation="Sezione di Partenza Parapetti Chicane"/>
+		<entry key="TrackElementName_ChicaneStartRailings" translation="Sezione di Partenza Chicane con guardrail"/>
 		<entry key="TrackElementName_ContractSection" translation="Partenza Tracciato Stretto"/>
-		<entry key="TrackElementName_ContractSectionRailings" translation="Inizio/Fine Parapetto Tracciato Stretto "/>
+		<entry key="TrackElementName_ContractSectionRailings" translation="Inizio/Fine Tracciato Stretto con guardrail"/>
 		<entry key="TrackElementName_ExpandSection" translation="Fine Tracciato Stretto"/>
-		<entry key="TrackElementName_ExpandSectionRailings" translation="Espandi Sezione Parapetto"/>
+		<entry key="TrackElementName_ExpandSectionRailings" translation="Espandi Sezione con guardrail"/>
 		<entry key="TrackElementName_NarrowTrack" translation="Stringi il Tracciato"/>
-		<entry key="TrackElementName_NarrowTrackRailings" translation="Stringi Binari del Tracciato"/>
-		<entry key="TrackElementName_NormalTrackRailings" translation="Binari"/>
+		<entry key="TrackElementName_NarrowTrackRailings" translation="Stringi Tracciato con guardrail"/>
+		<entry key="TrackElementName_NormalTrackRailings" translation="Guardrail"/>
```

Comment:
Perchè tipo da "Binari" a "Guardrail" mi sembra strano, soprattutto considerando che la key si chiama "TrackElementName_NormalTrackRailings"

Reply:
...Quindi ho tradotto in guardrail perchè ha più senso che parapetti o ringhiere..

Reply:
Manca ancora molto da tradurre, ma ho come l'impressione che il file su cui stiamo lavorando sia una versione piuttosto vecchia rispetto al gioco.. Infatti ho provato tutte le attrazioni per cercare questi maledetti railings ma ne ho trovati :/

Reply:
Infatti è così, il file su cui lavoriamo è della 1.3.1, è da un po' che non ne rilasciano un altro...

Reply:
Si, ho letto il tuo intervento sul forum Frontier.. ma per il momento nessuna risposta..
Speriamo pubblichino un nuovo file prima o poi.